packages/protocol/contracts/common/FixidityLib.sol

```diff
@@ -230,9 +226,9 @@ library FixidityLib {
   }
 
   /**
-   * @notice x/y. If the dividend is higher than the max fixed dividend, it
+   * @notice x/y. If the dividend is higher than the max dividend value, it
    * might overflow. You can use multiply(x,reciprocal(y)) instead.
-   * @dev The maximum value that can be safely used as a dividend is defined as
+   * @dev The maximum value that can be safely used as a dividend (maxFixedDividend) is defined as
```

Comment:
Looks like `maxFixedDividend` is the same as `maxNewFixed`, perhaps this can be used to simplify the comments.

Reply:
good point -- i'll update to just refer to it as maxNewFixed